### ISSUE TITLE
InvalidInlinteTag: `@see` javadoc false positive

### ISSUE THREAD

**Issue by lfgcampos** [google/error-prone#5140]
Error:
```
warning: [InvalidInlineTag] Tag name `see` is unknown. If this is a commonly-used custom tag, please click 'not useful' and file a bug.
```

Even though the `see` tag is present in the `JavadocTag` (see [here](https://github.com/google/error-prone/blob/master/core/src/main/java/com/google/errorprone/bugpatterns/javadoc/JavadocTag.java#L50)), while running errorprone, it complains it's not valid.

**Comment by graememorgan** [comment]
`@see` is a block tag not an inline one, unless there's a new version of it I'm not aware about.

**Comment by lfgcampos** [comment]
ah fair, my bad @graememorgan ... I haven't checked the `{~}` usage to identify it as a block vs inline

thanks for the quick reply though!


is there anything we can do to provide a better explanation (or even official links) when going to https://errorprone.info/bugpattern/InvalidInlineTag?
